test_runner/performance/test_perf_olap.py

```diff
@@ -17,6 +17,24 @@ class LabelledQuery:
     query: str
 
 
+# This must run before all tests in this module
+# create extension pg_stat_statements if it does not exist
+# and TEST_OLAP_COLLECT_PG_STAT_STATEMENTS is set to true (default false)
+# Theoretically this could be in a module or session scope fixture,
+# however the code depends on other fixtures that have function scope
+@pytest.mark.remote_cluster
+def test_clickbench_create_pg_stat_statements(remote_compare: RemoteCompare):
```

Comment:
There's one more thing that we can use here is `skipif`, then we don't need to have a condition in the function itself. 

```suggestion
@pytest.mark.skipif(os.getenv('TEST_OLAP_COLLECT_PG_STAT_STATEMENTS', 'false').lower() == 'true', reason="<put the reason here>")
def test_clickbench_create_pg_stat_statements(remote_compare: RemoteCompare):
```

And the same about `test_clickbench_collect_pg_stat_statements`

Reply:
OK, however afaik the logic is reversed:
```
@pytest.mark.skipif(os.getenv('TEST_OLAP_COLLECT_PG_STAT_STATEMENTS', 'false').lower() == 'false', reason="Skipping - Creating extension pg_stat_statements")
```